PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRawJetWithEP.h

```diff
@@ -1,88 +1,49 @@
-
-#ifndef AliAnalysisTaskRawJetWithEP_H
-#define AliAnalysisTaskRawJetWithEP_H
-/**
- * \file AliAnalysisTaskRawJetWithEP.h
- * \brief Declaration of class AliAnalysisTaskRawJetWithEP
- *
- * In this header file the class AliAnalysisTaskRawJetWithEP is declared.
- * This is a sample task that shows how to write a simple user analysis task
- * using the EMCal jet framework. It is also used to do automatic benchmark
- * tests of the software.
- *
- * \author Salvatore Aiola <salvatore.aiola@cern.ch>, Yale University
- * \date Apr 27, 2016
- */
-
-/* Copyright(c) 1998-2016, ALICE Experiment at CERN, All rights reserved. *
- * See cxx source for full Copyright notice                               */
-
+#ifndef ALIANALYSISTASKEMCALJETSAMPLE_H
+#define ALIANALYSISTASKEMCALJETSAMPLE_H
+// ******************************************************************************************
+// \class AliAnalysisTaskRawJetWithEP
+// \brief task used to load the Qn calibrations and get the calibrated Qn vectors for JE analyses
+// \authors:
+// C. Beattie, caitlin.beattie@cern.ch
+// M. Sas,     mike.sas@cern.ch
```

Comment:
Not as critical, but all of this is incorrect. It's better to have a correct description and correct author info

Reply:
Thank you for telling me that.  I modified these lines.